scripts/check_toc_is_complete.py

```diff
@@ -5,16 +5,16 @@
 You can run it manually with `pre-commit run check-toc --all`.
 """
 
-import json
 from pathlib import Path
 import argparse
+import ast
```

Comment:
these aren't sorted, but their order will be sorted out with #4239 (which needs to have upstream/master merged anyway)